test/hooks/index.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const { BadRequest } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)